### PR TITLE
Alekhya | MOBN-794 | Popup for click on admin, manage appointments and back button

### PR DESCRIPTION
**Pull request by AlekhyaYalla**

### CODE REVIEW

File: ui/react-components/containers/AppointmentContainer.jsx [hunk 1]
```diff
@@ -2,45 +2,32 @@ import React, {Component} from "react";
...
         super(props);
         this.state = {
             locale: getLocale(),
             messages: translations[props.locale],
             appConfig: null
         };
         (async () => {
-            await this.getMessages();
-            await this.getAppConfigs();
+            this.setState({messages: await getMessages(this.state.locale)});
```
Comment (bsneha90) [inline ui/react-components/containers/AppointmentContainer.jsx]: can we use react hooks for state?

File: ui/reactAngularAdaptor.js [hunk 2]
```diff
@@ -1,24 +1,53 @@
...
     $scope.onBack = function () {
-        $state.go('^', $state.params, {reload: true});
+        onBack = true;
+        $state.go(backUrl, $state.params, {reload: true});
+        ngDialog.close();
     };
 
+    $state.showCancelConfirmation = false;
+    $scope.$on('$stateChangeStart', function (event, next, current) {
+        if (next.url !== "/new" && !onBack) {
```
Comment (bsneha90) [inline ui/reactAngularAdaptor.js]: can you check if it works for edit as well?

Comment (AlekhyaYalla) [reply ui/reactAngularAdaptor.js]: Okay will check

File: ui/react-components/containers/AppointmentContainer.jsx (hunk 1 above)
Comment (AlekhyaYalla) [reply ui/react-components/containers/AppointmentContainer.jsx]: I dont think react hooks can be used in class component.

File: ui/reactAngularAdaptor.js (hunk 2 above)
Comment (AlekhyaYalla) [reply ui/reactAngularAdaptor.js]: Working fine with edit as well. @bsneha90

**Pull request merged**